scipy/special/tests/test_ndtri_exp.py

```diff
@@ -60,13 +60,32 @@ def test_in_interval(self, interval, expected_rtol, uniform_random_points):
         )
 
     def test_extreme(self):
-        assert_func_equal(
-            log_ndtr_ndtri_exp,
-            lambda y: y,
-            [-np.finfo(float).max, -np.finfo(float).min],
-            rtol=1e-12,
-            nan_ok=True
-        )
+        # bigneg is not quite the largest negative double precision value.
```

Comment:
This is a really good consideration. I wrote this test kind of as an after-thought and didn't pay sufficient attention that everything actually made sense. Sigh, that `nan_ok` is practically unforgivable. Thanks for the in-depth explanation.